recipes/fakeit/all/test_package/example.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+#include <string>
+#include <standalone/fakeit.hpp>
```

Comment:
Does this have to be prefixed with `standalone/`? Given that the examples in the main repo are just `#include <fakeit.hpp>` (and I assume that it how most people already include it), can it be changed?

Reply:
By default the package doesn't set a default integration, so you have to explicitly name it when including the header. You can use `#include <fakeit.hpp>` if you set the package option `integration` to something other than `None`.